src/Microsoft.Health.Fhir.R4.Core/Features/Conformance/ListedCapabilityStatement.cs

```diff
@@ -0,0 +1,72 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using EnsureThat;
+using Hl7.Fhir.Model;
+using Microsoft.Health.Fhir.Core.Extensions;
+using static Hl7.Fhir.Model.CapabilityStatement;
+
+namespace Microsoft.Health.Fhir.Core.Features.Conformance
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "This is a DTO-style class")]
+    public sealed class ListedCapabilityStatement : IListedCapabilityStatement
```

Comment:
Could we have a base version of this and specific implementations?

Reply:
I'll do this in a separate PR